jsonnet/versions.yaml

```diff
@@ -15,7 +15,7 @@ repos:
     thanos: openshift/thanos
 versions:
     alertmanager: 0.27.0
-    kubeRbacProxy: 0.18.1
+    kubeRbacProxy: 0.18.2
```

Comment:
could you revert this? as it's not related to the PR. (in case the PR is reverted we don't want this to get reverted e.g.)
We'll take case of it in another PR. `versions` job isn't required you can keep it failing.

Reply:
No need to worry about the `versions` ci job. It just informative for us and not a required check.

Reply:
Fxied